packages/core/src/toggle/entrypoint.tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../tsconfig.lib.json",
+  "compilerOptions": {
+    "composite": true
+  },
+  "references": [{ "path": "../internal/entrypoint.tsconfig.json" }, { "path": "../checkbox/entrypoint.tsconfig.json" }]
```

Comment:
Forms?

Also why are we pulling in checkbox? It doesn't look like `toggle.element.ts` or `register.ts` make any reference to it...